### PR TITLE
conda-pack windows env

### PR DESCRIPTION
**Pull request by strseb**
## Description

We're currently fetching the whole conda-build env from the internet on every build. 
For android, we have moved that all into a toolchain task, allowing us to cache that, making stuff more stable :) 

Let's do that for the others too.

### CODE REVIEW

File: taskcluster/ci/toolchain/conda.yml [hunk 1]
```diff
@@ -5,64 +5,24 @@
 task-defaults:
     description: "Conda Pack an enviroment"
```
Comment (vinocher) [inline taskcluster/ci/toolchain/conda.yml:6]: Should conda.yml be renamed to conda_windows.yml to match the naming pattern of conda_android.yml? Consider if the description here should say 'windows environment', and if the description in conda_android.yml should be similarly updated. (Nit: typo in 'environment'.)

File: scripts/windows/conda_trim.ps1 [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
```
Comment (vinocher) [inline scripts/windows/conda_trim.ps1:5]: Consider adding a comment explaining what this file does.

File: scripts/windows/conda_trim.ps1 [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+$X_WIN_VERSION = "0.2.10"
+
+$conda_env = conda info --json | ConvertFrom-Json
+$OLD_PWD = $PWD # Backup that to go back once we done.
```
Comment (vinocher) [inline scripts/windows/conda_trim.ps1:9]: Does OLD_PWD need to be set here? Just checking if is used.

File: scripts/windows/conda_trim.ps1 [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
```
Comment (vinocher) [inline scripts/windows/conda_trim.ps1:4]: Is conda_trim.ps1 being called? If so, for my understanding, could you please point to where it is called?

Comment (strseb) [reply scripts/windows/conda_trim.ps1:4]: Oh sorry. I was initially concerned about the bundle size, so i wanted to trim not required packaged. However currently the fetch performance is fine so i never actually added anything to remove.

File: taskcluster/ci/toolchain/conda.yml (hunk 1 above)
Comment (strseb) [reply taskcluster/ci/toolchain/conda.yml]: Fixed the typo, and moved that to a `conda_windows.yml` good callout thanks!

**Review by vinocher — APPROVED** [review]

**Pull request merged**